<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>voting_window</class>
 <widget class="QMainWindow" name="voting_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="summary_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Voting Count</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Count</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Voting for Candidates</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_enter">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="id_label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Your ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="candidate_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Vote one candidate</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="john_button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>John</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="jane_button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Jane</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="other_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Other:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="other_lineedit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm Vote</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
